charts/pega/charts/constellation/templates/clln-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+#Deploy only when the constellation flag has been enabled in the values yaml.
+{{ if and .Values.enabled (eq .Values.enabled true) }}
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: constellation
+  labels:
+    app: constellation
+    # component: constellation
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: constellation
+      # component: constellation
+  template:
+    metadata:
+      labels:
+        app: constellation
+        # component: constellation
+    spec:
+      imagePullSecrets:
+      - name: {{ template "pegaRegistrySecret" }}
+      containers:
+      - name: constellation
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        image: {{ .Values.image }}
+        args:
+        - port=3000
+        - {{ .Values.logLevel }}
+        livenessProbe:
```

Comment:
Maybe add readiness probe as well with the same endpoint?

Reply:
added